apps/avifgainmaputil/convert_command.cc

```diff
@@ -64,7 +64,19 @@ avifResult ConvertCommand::Run() {
     // If there is no ICC and no CICP, assume sRGB by default.
     image->colorPrimaries = AVIF_COLOR_PRIMARIES_SRGB;
     image->transferCharacteristics = AVIF_TRANSFER_CHARACTERISTICS_SRGB;
-    image->gainMap->altColorPrimaries = AVIF_COLOR_PRIMARIES_SRGB;
+    if (image->gainMap && !image->gainMap->altICC.size) {
```

Comment:
Nit/optional: Use the same form (`size == 0` or `!size`) to test if ICC size is 0 at line 60 and here.